web/pdf_page_view.js

```diff
@@ -260,6 +260,10 @@ class PDFPageView {
    */
   updatePosition() {
     if (this.textLayer) {
+      let readableStream = this.pdfPage.streamTextContent({
+        normalizeWhitespace: true,
+      });
+      this.textLayer.setTextContentStream(readableStream);
```

Comment:
We just re-requesting data here since we are not storing entire text content anymore. getReader() can be requested once. We can cache entire textContent content after the read, however I want to try to get away without caching.